scripts/check-binaries.sh

```diff
@@ -15,7 +15,7 @@ for BINARY in "${BINARIES[@]}"; do
     test_command "${BINARY}-${LLVM_VERSION}"
 done
 
-QEMU_SUFFIXES=( "arm" "aarch64" "i386" "mips" "mipsel" "ppc" "ppc64" "x86_64" )
+QEMU_SUFFIXES=( "arm" "aarch64" "i386" "mips" "mipsel" "ppc" "ppc64" "riscv32" "riscv64" "x86_64" )
```

Comment:
Since `qemu-system-misc` also contains `qemu-system-s390x`, we can add it to this list as well.

Reply:
done